submissions/unabyband/popup-task/main.css

```diff
@@ -0,0 +1,345 @@
+  border-color: transparent;
+  border-bottom: solid #fff;
+  border-style: dashed dashed solid;
+  border-width: 0 8.5px 8.5px;
+  right: 102.5px;
+  top: -9px;
+}
+
+/*Кнопка відкриття попапа*/
+.button_popup {
+    width: 30px;
+    height: 30px;
+    display: block;
+    background: url("../icons/popup-button.png") center center no-repeat;
+    cursor: pointer;
+    border: 1px solid transparent;
+}
+/*Фокусування на чекбоксі та кнопці відкриття попапа*/
+.input_popup:focus ~ .button_popup {
+    border: 1px solid lightgray;
+    border-radius: 10%;
+}
+/*Рамка активації кнопки відкриття попапа*/
+.input_popup:checked ~ .button_popup{
+    border: 2px solid rgba(0, 0, 0, 0.75);
+    border-radius: 10%;
+}
+/*Правило для класу попапа*/
+.popup_window {
+    width: 100%;
+    height: 100%;
+    overflow: auto;
+}
+/*Загальний клас елементів попапа*/
+.popup_window_list {
+    display: flex;
+    flex-wrap: wrap;
+    padding: 28px 18px 28px 28px;
+}
+/*Підклас видимих елементів попапа*/
+.popup_window_visible {
+    margin-top: 0px;
+    padding-top: 0px;
+}
+/*Підклас прихованих елементів попапа*/
+.popup_window_hidden {
+    display: none;
+}
+/*Контейнер елемента попапа*/
+.popup_window_list__item {
+    width: 80px;
+    height: 80px;
+    display: flex;
+    justify-content: center;
+    margin-top: 10px;
+}
+/*Поведінка елементів попапа*/
+.popup_window_list__item:hover {
+    border: 1px solid rgba(0, 0, 0, 0.75);
+}
+/*Зображення елемента попапа*/
+.popup_window__img {
+    height: 60px;
+    width: 60px;
+    display: block;
+}
+/*Підпис елемента попапа*/
+.popup_window__label {
+    font-size: 13px;
+    text-align: center;
+    color: rgba(0, 0, 0, 0.87);
+    text-decoration: none;
+    display: block;
+}
+/*Кінець елементів попапа - лінк Even More*/
+.top_menu__item_other {
+    display: block;
+    flex: auto;
+    text-align: center;
+    padding-top: 30px;
+    margin-bottom: -35px;
+}
+/*---------------------------------------------------------------------------
+Чекбокс кнопки More*/
+.hidden_input {
+    appearance: none;
+    box-sizing:content-box;
+    margin: 0px;
+    border: none;
+}
+/*Фокусування на чекбоксі та кнопці More*/
+.hidden_input:focus ~ .morebutton {
+    border: 1px solid rgba(0, 0, 0, 0.75);
+    border-radius: 3%;
+}
+/*Кнопка More*/
+.morebutton {
+    display: block;
+    flex: auto;
+    text-align: center;
+    background-color: rgba(128, 128, 128, 0.1);
+    padding: 10px 0px 9px 0px;
+    margin-top: -24px;
+    cursor: pointer;
+}
+/*Відкриття прихованих елементів попапа*/
+.hidden_input:checked ~ .popup_window_hidden {
+    border-top: 2px solid #e5e5e5;
+    display: flex;
+}
+/*Приховання кнопки More при натисканні*/
+.hidden_input:checked ~ .morebutton {
+    display: none;
+}
+/*---------------------------------------------------------------------------
+Прив'язка зображень іконок до видимих елементів попапа*/
+.popup_window_visible .popup_window_list__item:first-child .popup_window__img {
```

Comment:
Semantically this is not the correct use of images, this is not a background in this case, but icons. So, you need to move it to an HTML file as `img` tags

Reply:
Done! 
Plus made a  few Photoshop manipulations over icons files in order to make them square.